apps/src/code-studio/pd/regional_partner_mini_contact/RegionalPartnerMiniContact.jsx

```diff
@@ -0,0 +1,141 @@
+      notes: PropTypes.string
+    }),
+    apiEndpoint: PropTypes.string
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      submitting: false,
+      submitted: false,
+      errors: [],
+      name: this.props.options.user_name,
+      email: this.props.options.email,
+      zip: this.props.options.zip,
+      notes: this.props.options.notes
+    };
+  }
+
+  submit = () => {
+    const params = {
+      name: this.state.name,
+      email: this.state.email,
+      zip: this.state.zip,
+      notes: this.state.notes
+    };
+
+    this.setState({submitting: true});
+
+    this.submitRequest = $.ajax({
+      method: 'POST',
+      url: this.props.apiEndpoint,
+      contentType: 'application/json',
+      data: JSON.stringify({form_data: params}),
+      complete: result => {
+        this.onSubmitComplete(result);
+      }
+    });
+  };
+
+  handleChange = change => {
+    this.setState(change);
+  };
+
+  onSubmitComplete = results => {
+    if (
+      results.responseJSON &&
+      results.responseJSON.errors &&
+      results.responseJSON.errors.form_data
+    ) {
+      if (results.responseJSON.errors.form_data) {
+        this.setState({
+          errors: results.responseJSON.errors.form_data,
+          submitting: false
+        });
+      }
+    } else if (results.responseJSON) {
+      this.setState({submitted: true, submitting: false});
+    } else {
+      this.setState({submitted: false, submitting: false});
+    }
+  };
+
+  render() {
+    if (this.state.submitted) {
+      return (
+        <div>Your message has been sent. Thank you. We'll be in touch.</div>
+      );
+    } else {
+      return (
+        <FormGroup>
+          <p>
+            If you're interested in Professional Learning, just fill out this
+            form and a local Regional Partner will be in touch!
+          </p>
+          <FieldGroup
+            id="name"
+            label="Name"
+            type="text"
+            onChange={this.handleChange}
+            defaultValue={this.state.name}
+          />
+          {this.state.errors.includes('email') && (
+            <div style={styles.error}>Please enter an email.</div>
+          )}
+          <FieldGroup
+            id="email"
+            label="Email"
+            type="text"
+            required={true}
+            onChange={this.handleChange}
+            defaultValue={this.state.email}
+          />
+          {this.state.errors.includes('zip') && (
+            <div style={styles.error}>Please enter a ZIP code.</div>
+          )}
+          <FieldGroup
+            id="zip"
+            label="Zip"
+            type="text"
+            required={true}
+            onChange={this.handleChange}
+            defaultValue={this.state.zip}
+          />
+          <FieldGroup
+            id="notes"
+            label="Questions or notes for your local Regional Partner"
+            type="text"
+            componentClass="textarea"
```

Comment:
This is where user enters question which could be in form of a short email. Should we make the text area bigger?

Reply:
Good question.  We can also address this in a UI pass soon.  The control is responsive and so it might depend on how big we want it to be overall.  The user can also resize it manually already.